src/api.ts

```diff
@@ -102,7 +102,12 @@ export class JamsocketApi {
 
   public getLoginUrl(): string {
     const hostname = new URL(this.apiBase).hostname
-    return `https://app.${hostname}/cli-login`
+    const parts = hostname.split('.')
+    if (parts.length > 2) {
```

Comment:
This strikes me as a special case trying to be a general case; I'm not sure what other hostname besides `api.blah.blah` would apply here. I'm inclined to say we should either special case `api.jamsocket.com`, or special-case `api.*`

Reply:
Pushed an update - let me know what you think!